LTRegistratorApi/LTRegistratorApi/Controllers/AccountController.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
```

Comment:
Can I ask you to introduce:
1) Summary to class, fields, properties and methods.
2) Regions for: public methods, private methods, public fields, private fields...

#region private methods
//some code with private methods
#endregion